src/Files.App/ServicesImplementation/VolumeInfoFactory.cs

```diff
@@ -18,25 +15,7 @@ public async Task<VolumeInfo> BuildVolumeInfo(string driveName)
 
         private async Task<string> GetVolumeID(string driveName)
```

Comment:
The `async Task<string>` here is redundant. Can we change it to a sync method?

Reply:
> Can we change it to a non-async method which simply returns a string?

Done!